lib/omnibus/compressors/dmg.rb

```diff
@@ -134,63 +108,33 @@ def clean_disks
     end
 
     #
-    # Create a writable dmg we can put assets on.
+    # Create a compressed dmg.
     #
-    def create_writable_dmg
-      log.info(log_key) { "Creating writable dmg" }
+    def create_compressed_dmg
+      log.info(log_key) { "Creating compressed dmg" }
 
       shellout! <<-EOH.gsub(/^ {8}/, "")
-        hdiutil create \\
-          -volname "#{volume_name}" \\
-          -fs HFS+ \\
-          -fsargs "-c c=64,a=16,e=16" \\
-          -size 512000k \\
-          "#{writable_dmg}" \\
-          -puppetstrings
+        pip install dmgbuild==1.6.5
```

Comment:
`omnibus` doesn't seem to have proper Python deps: should I move this one somewhere else?